CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# How to contribute
+
+Support and contributons from the open source community are essential for keeping
+MEAN.js up to date and always improving! There are a few guildlines that we need
+contributors to follow to keep the project consistent, as well as allow us to keep
+maintaining MEAN.js in a reasonable amount of time.
+
+## Getting Started
+
+Before you begin making changes:
+* Submit an Issue on Github, assuming one does not already exist.
+* Clearly describe the issue, including the steps to reproduce the issue.
+	* If it's a new feature, enhancement, or restructure, Explain your reasonings on why you think it should be added, as well as a particular use case.
+
+## Making Changes
+
+* Create a topic branch from the master branch.
+* Check for unnecessary whitespace / changes with `git diff --check` before committing.
+	* Also check that your code is formatted properly with spaces (hint: Use .editorconfig)
+* Keep git commit messages clear and appropriate
+	* If possible, please "squash" your commits to as few commits as possible/reasonable.
```

Comment:
I would make it more specific go squash into reasonable separations such as one commit for implementation, one for tests, one for documentation before finally squashing to one commit when getting the LGTM from a collaborator.